mathics/builtin/statistics/dependency.py

```diff
@@ -20,7 +20,11 @@
 
 SymbolConjugate = Symbol("Conjugate")
 SymbolCovariance = Symbol("Covariance")
+
+# Something is weird here. No System`. And we can't use what is in
+# SymbolSqrt from systemsymbols?
```

Comment:
Indeed, is the same symbol. Please just refer it from `mathics.core.systemsymbols`

Reply:
I had tried replacing this and got an error. (I did not save nor do I remember the error). 

I think what is different here is that instances of SqrtBox have some sort of dimension information and somehow (but I don't know why or how) that makes a difference.